tools/elk/lib/main.sc

```diff
@@ -158,7 +158,7 @@ def generateLogstashConfig(inputPath: Path, targetPath: Path, logFormat: LogForm
       "FILE" -> util.escapeString(logPath.toString),
       "SINCEDB" -> util.escapeString((targetPath / s"since-db-${host}.db").toString),
       "CODEC" -> logFormat.codec,
-      "EXTRA" -> s"""|"add_field" => {
+      "EXTRA" -> s"""|add_field => {
```

Comment:
new logstash suddenly considered it invalid to have quotes around add_field. Tada! Breaking compatibility.